spec/services/unsent_documents_service_spec.rb

```diff
@@ -88,7 +88,7 @@
 
       it "sets the Raven exception context to the ticket ID when operating on the ticket ID" do
         service.detect_unsent_docs_and_notify
-        expect(service).to have_received(:with_raven_context).with(hash_including(ticket_id: 1))
+        expect(service).not_to have_received(:with_raven_context).with(hash_including(ticket_id: 1))
```

Comment:
This intake is no longer fed to the each loop! Because the its docs have already been sent, it is not included in the query.